packages/mermaid/src/diagrams/info/infoTypes.ts

```diff
@@ -0,0 +1,10 @@
+import type { DiagramDb } from '../../diagram-api/types.js';
+
+/**
+ * Info diagram DB.
+ */
```

Comment:
You can remove comments like these that don't really explain anything, good names are enough. 
`Readable Names >>> Comments`.

Reply:
I left it there as a placeholder; I wasn't sure what to add more, so I left it like that, and yes, readable names are more than enough, but I guess we need to add some to DB functions even if the name was clear since they're kind of fuzzy, especially for those that have more than one or even object parameters.